### PR TITLE
add api roundtrip

### LINKED ISSUE #4: add test for webxdc apis
to test if webxdc apis are available and work

### PR DESCRIPTION
**Pull request by r10s**
basic test of sending/receiving updates.

at some point, we should make the css a bit nicer and move failed test atop and/or be less wordy if things are okay.

closed #4

### CODE REVIEW

File: js/update-api.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+window.addEventListener("load", () => {
+
+    var updatesInitialized = false
+    var previousUpdates = 0
+    var currentUpdates = 0
```
Comment (adbenitez) [inline js/update-api.js:5]:
```suggestion
    var updatesInitialized = false;
    var previousUpdates = 0;
    var currentUpdates = 0;
```

File: js/update-api.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          h("span", {}, "updates from previous/current runs: "),
+          h("strong", {id: "previous-runs"}, "0"),
+          h("span", {}, "/"),
+          h("strong", {id: "current-run"}, "0"),
+         )
+    );
+
+    window.webxdc.setUpdateListener(function (update) {
+        if (!updatesInitialized) {
+            previousUpdates++
```
Comment (adbenitez) [inline js/update-api.js:21]:
```suggestion
            previousUpdates++;
```

File: js/update-api.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        } else {
+            currentUpdates++
```
Comment (adbenitez) [inline js/update-api.js:23]:
```suggestion
            currentUpdates++;
```

File: js/update-api.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+            document.getElementById("updates-error").style.display = "none";
+            document.getElementById("updates-ok").style.display = "inline";
+        }
+        document.getElementById("previous-runs").innerHTML = previousUpdates;
+        document.getElementById("current-run").innerHTML = currentUpdates;
+    }).then(() => {
+        updatesInitialized = true
```
Comment (adbenitez) [inline js/update-api.js:30]:
```suggestion
        updatesInitialized = true;
```

File: js/update-api.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        window.webxdc.sendUpdate({payload: { foo: "bar"}}, "test update");
```
Comment (adbenitez) [inline js/update-api.js:31]:
```suggestion
        window.webxdc.sendUpdate({payload: { "update-api-test": "bar"}}, "test update");
```

**Review by adbenitez — APPROVED** [review]

File: js/update-api.js (hunk 5 above)
Comment (r10s) [reply js/update-api.js:31]: :)

File: js/update-api.js (hunk 1 above)
Comment (r10s) [reply js/update-api.js:5]: yeah, let's go for the semicolon;)

**Pull request merged**